### ISSUE TITLE
Name Property for Edit box in 'preferences' window is not defined 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#361]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to 'Tools' menu item present in menu bar and Expand it
3. Navigate to 'Preferences' menu item and select it
4. Navigate to 'general' tab item and select it
5. Navigate to Edit box and observe Inspect property

### Expected Behavior
Name property should be defined properly for edit box (For e.g. 'Font-Size')
Note:
'RangeValue.Maximum' and  'RangeValue.Minimum' should be properly defined as 72 and 9 Respectively

### Actual Behavior
Name property is not defined for Edit box 
'RangeValue.Maximum' and  'RangeValue.Minimum' is not defined
**Narrator Behavior:** Narrator is announcing as 'UI, editing, 72,0% vertically scrolled '
**NVDA Behavior:** NVDA is announcing as 'UI grouping,edit, 72'

### User Impact
Users who rely on screen reader will not be able to know the Name of edit box and hence will not know the purpose of edit box

### Screenshots
![edit_name_property_inspect](https://user-images.githubusercontent.com/22956130/40724850-ae410776-643f-11e8-8392-54d694b687d1.png)
![edit_name_property_narrator](https://user-images.githubusercontent.com/22956130/40724856-af687350-643f-11e8-8d89-ff97a98b4a7c.png)
![edit_name_property_nvda](https://user-images.githubusercontent.com/22956130/40724869-b3a5db24-643f-11e8-873e-23013e725af4.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1
